### PR TITLE
feat: Add Grand Theft Auto V - RAGE MP support

### PR DESCRIPTION
**Pull request by xCausxn**
Implements support for getting server info from the rage mp server list.

Completes: #49

### CODE REVIEW

**Comment by xCausxn** [comment]
@CosminPerRam I opted to just return the entry server result entry as the "raw" field. Partly due to the fact potential info for the nested entry is available at the top level.

We could however remove excess entries and move the servers[X] entry to the top level for our case.

**Review by CosminPerRam — APPROVED** [review]
Great stuff!

**Comment by CosminPerRam** [comment]
> Partly due to the fact potential info for the nested entry is available at the top level.

We can leave it this way.

File: CHANGELOG.md [hunk 1]
```diff
@@ -392,4 +393,4 @@ during the query.
 
 ### 1.0.0
 * First official release
-* Node.js 6 is now required
+* Node.js 6 is now required
```
Comment (CosminPerRam) [inline CHANGELOG.md:396]: Hmmm, seems like a stray change here?

File: lib/games.js [hunk 2]
```diff
@@ -1270,6 +1270,14 @@ export const games = {
...
     }
   },
+  gta5r: {
+    name: 'Grand Theft Auto V - RageMP',
+    release_year: 2016,
+    options: {
+      port: 22005,
+      protocol: 'ragemp'
+    }
+  },
```
Comment (CosminPerRam) [inline lib/games.js:1280]: Please add this to properly make the table generator reference the note:
```
extra: {
  doc_notes: 'gta5r'
}
```

Like as in:
```
  trackmaniaforever: {
    name: 'Trackmania Forever',
    release_year: 2008,
    options: {
      port: 2350,
      port_query: 5000,
      protocol: 'nadeo'
    },
    extra: {
      doc_notes: 'nadeo'
    }
  },
  ```
```

**Review by CosminPerRam — CHANGES_REQUESTED** [review]

**Review by CosminPerRam — APPROVED** [review]

**Pull request merged**